src/lib/containers/GenericCard.tsx

```diff
@@ -399,7 +403,7 @@ export default class GenericCard extends React.Component<
       // undefined, take default value from class
       marginTop: isHeader ? '0px' : undefined,
       marginBottom: isHeader ? '0px' : undefined,
-      paddingBottom: showFooter ? undefined : '15px',
+      paddingBottom: showFooter || imageFileHandleIdValue ? undefined : '15px',
```

Comment:
Set the padding bottom to 15, unless we're showing a footer or an image on the left.